src/IotCentral/IotCentral/IotCentral/NewAzureRmIotCentralApp.cs

```diff
@@ -112,7 +112,14 @@ public override void ExecuteCmdlet()
 
         private string GetAppSkuName()
         {
-            return this.Sku ?? PSIotCentralAppSku.S1.ToString();
+            string skuName;
+            if ((object)this.Sku != null && (int)this.Sku != 0) {
+                skuName = this.Sku.ToString();
+            } else {
+                skuName = PSIotCentralAppSku.S1.ToString();
```

Comment:
This should default to "ST2" sku

Reply:
I will be changing the default sku for all the SDKs later on. so probably fine here. I think all of the SDKs for iotc are default to S1 right now.